lib/constants.js

```diff
@@ -6,6 +6,7 @@ export const FONTS = [
   { id: 'Fantasque Sans Mono', name: 'Fantasque Sans Mono' },
   { id: 'Monoid', name: 'Monoid' },
   { id: 'Fira Code', name: 'Fira Code' },
+  { id: 'IBM Plex Mono', name: 'IBM Plex Mono' },
```

Comment:
Thanks @ericadamski can you move this to be in alphabetical order? Thanks (:

Reply:
For sure @mfix22!